src/core/utils.js

```diff
@@ -153,3 +157,17 @@ export function browserBase64Decode(str) {
     return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
   }).join(''));
 }
+
+export function apiAddonType(addonType) {
+  if (!API_ADDON_TYPES_MAPPING[addonType]) {
```

Comment:
This should probably use `hasOwnProperty()` or use a `Map` or be an object created with `Object.create(null)`. Currently I believe this test would fail:

```js
assert.throws(() => {
  apiAddonType('hasOwnProperty'); // This would return a function
});
```

http://localhost:3000/en-US/firefox/hasOwnProperty/ ~> 500 error